Version 5 (#7515)
apollographql/apollo-server: docs/source/migration.mdx

### Node.js

Apollo Server 5 supports Node.js 18.0.0 and later. (Apollo Server 4 supports Node.js 14.16.0.) This includes all LTS and Current major versions at the time of release.

If you're using Node.js 14, upgrade your runtime before upgrading to Apollo Server 5.

> If upgrading to Node.js 18 is not possible, you can upgrade to or continue to use Apollo Server 4 without issue.

Reviewer: ```suggestion
If you can't upgrade to Node.js 18, you can use Apollo Server 4 without issue.
```
